primer/brand
# Add new Grid component

## Summary

Adds a 12 column, responsive grid component for flexible layout.

🔗 [See documentation](https://primer-89832c6e10-26139705.drafts.github.io/components/Grid)
🔗 [See Storybook examples](https://primer-89832c6e10-26139705.drafts.github.io/storybook/?path=/story/components-grid--default)

## List of notable changes:



- **added** base Grid component to match Figma counterpart
- **added** prop for `enableOverlays` for visual aid

## What should reviewers focus on?

- Review [documentation](https://primer-89832c6e10-26139705.drafts.github.io/components/Grid) and verify the examples are clear and comprehensive
- Review API 
- Try it out in the docs sandbox

## Steps to test:



1. Open the Grid component documentation and/or storybook
2. Verify that Grid works as expected

## Supporting resources (related issues, external links, etc):

- [Initial API exploration](https://www.figma.com/file/M4gYzqZSDHVBZNSzRcx9Ej/Primer-Brand-Grid-API-exploration?node-id=0%3A1&t=ahUTqblluwUODzmH-1)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

![Grid in 12 column arrangement](https://user-images.githubusercontent.com/13340707/233014249-414b7376-2660-43e6-b8dc-d6d895864185.png)

![Grid in asymmetrical arrangement](https://user-images.githubusercontent.com/13340707/233014289-fec5cb18-c703-4e81-bfcf-99ed20b543d6.png)
